docs/conf.py

```diff
@@ -156,6 +158,44 @@
     "dudir": ("http://docutils.sourceforge.net/docs/ref/rst/directives.html#%s", ""),
 }
 
+clang_format_signatures_domain_styles = {
+    "cpp": """{
+    BasedOnStyle: LLVM,
+    ColumnLimit: 70,
+    }""",
+}
+
+html_wrap_signatures_with_css = ["py"]
+
+external_cpp_references = {
+    "nlohmann::json": {
+        "url": "https://json.nlohmann.me/api/json/",
+        "object_type": "type alias",
+        "desc": "C++ type alias",
+    },
+    "nlohmann::basic_json": {
+        "url": "https://json.nlohmann.me/api/basic_json/",
+        "object_type": "class",
+        "desc": "C++ class",
+    },
+}
+
+cpp_strip_namespaces_from_signatures = ["my_ns1", "my_ns2"]
```

Comment:
Same idea here. Specify the extension that this is related to and optionally use a `literalinclude` block in the docs.

Reply:
Done.